app/src/NativeBase/Components/TopOfApp.tsx

```diff
@@ -39,7 +41,14 @@ const TopOfApp: FC<TopOfAppProps> = ({
 
   return (
     <>
-      <StatusBar barStyle={statusBarStyle} />
+      <StatusBar
+        backgroundColor={
```

Comment:
The `StatusBar` component won't let us use `_dark` and `_light` in the same way e.g. `Box` will below (which is a more NativeBase way of doing it)